fostyler/fos-flagship
# Feature/change form success behavior

Have implemented the 3 changes above.
For the following issue: https://github.com/fostyler/fos-flagship/issues/4

To test: 
1. open homepage
2. fill out the form including all required fields and click submit. 
3. Notice that the submit button will be disabled while the form is being submitted
4. if the submission is successful, the form will be cleared upon completion and a success message will display in a box, which disappears after 4 seconds.

@LailaAli @fostyler
